src/apps/demo-app/src/app/app.module.ts

```diff
@@ -1,19 +1,12 @@
 import {BrowserModule} from '@angular/platform-browser';
 import {NgModule} from '@angular/core';
 import {BrowserAnimationsModule} from '@angular/platform-browser/animations';
-import {FlexLayoutModule, BREAKPOINT} from '@angular/flex-layout';
+import {FlexLayoutModule} from '@angular/flex-layout';
 
 import {RoutingModule} from './routing.module';
 import {AppComponent} from './app.component';
 import {DemoMaterialModule} from './material.module';
 
-const PRINT_BREAKPOINTS = [{
```

Comment:
Please change this to a non-print custom breakpoint since this tests if it AOT compiles with them.

Reply:
Since this code has been removed, what are you saying?

Reply:
I'm saying _don't_ remove it, because it's actually testing something important (probably deserves a comment to that effect too). So instead it can either be included as-is, or it can be changed to another custom breakpoint (e.g. xxs).